### PR TITLE
Big update

### PR DESCRIPTION
**Pull request by cboiteux2765**
Added long description and event expanded image after clicking card (CSS still in progress), solved card alignment issue for board members, updated club info such as tech dev position names and FAQs

### CODE REVIEW

**Review by WinbertZhang — CHANGES_REQUESTED** [review]
For the event cards, the flip side images sometimes take a while to load and the animation after clicking on the event can be clunky. If possible we can try to instead have each card link to a separate page that contains the full description instead

Looks like register is currently down, let’s keep it out of the pull request for now until the page is up

File: data/alumni.ts [hunk 1]
```diff
@@ -1,143 +1,211 @@
...
     imageUrl: "/images/alumni/ishana-narayanan.jpg",
+    linkedIn: "https://www.linkedin.com/in/ishana-n-58a90b170/"
   },
   {
     name: "Brian Lim",
     gradYear: "UCSB '21",
-    description:
-      "Brian studied computer science at UCSB and is now a software development engineer at Amazon.",
+    major: "Computer Science",
+    job: "Software Engineer @ Amazon",
```
Comment (BLimmie) [inline data/alumni.ts:102]: Should be Palantir

**Comment by BLimmie** [comment]
Maybe something to look into for a future PR: pulling directly from Linkedin the latest job from each alumni so this doesn't need manual changing

**Comment by WinbertZhang** [comment]
> Maybe something to look into for a future PR: pulling directly from Linkedin the latest job from each alumni so this doesn't need manual changing

Moved to Issue #4!

**Review by WinbertZhang — APPROVED** [review]
LGTM

**Pull request merged**